cli/command/container/opts.go

```diff
@@ -236,9 +236,12 @@ func addFlags(flags *pflag.FlagSet) *containerOptions {
 	flags.Int64Var(&copts.cpuPeriod, "cpu-period", 0, "Limit CPU CFS (Completely Fair Scheduler) period")
 	flags.Int64Var(&copts.cpuQuota, "cpu-quota", 0, "Limit CPU CFS (Completely Fair Scheduler) quota")
 	flags.Int64Var(&copts.cpuRealtimePeriod, "cpu-rt-period", 0, "Limit CPU real-time period in microseconds")
+	flags.SetAnnotation("cpu-rt-period", "version", []string{"1.25"})
```

Comment:
Wondering if we should add a utility for this, or to specify the version when adding flags (this does the job, but it's rather ugly to have these `SetAnnotation()` everywhere.